ui/components/multichain/address-list-item/index.scss

```diff
@@ -1,9 +1,4 @@
 .address-list-item {
-  /* Set explicitly to match the font-size of AccountListItem */
-  &__label {
-    font-size: 14px !important;
-  }
```

Comment:
Ensuring the address in account list item adheres to the Text variant that uses `bodyMdMedium`